nav2_behavior_tree/include/nav2_behavior_tree/compute_control_action.hpp

```diff
@@ -58,10 +59,11 @@ class FollowPathAction : public BtActionNode<nav2_msgs::action::FollowPath>
   {
     return {
       BT::InputPort<nav_msgs::msg::Path>("path", "Path to follow"),
+      BT::InputPort<std::string>("controller_name", "FollowPath"),
```

Comment:
... and link them together: `BT::InputPort<std::string>("motion_type", "FollowPath"),`